### ISSUE TITLE
"transform styles SVG elements" feature needs to be detected

### ISSUE THREAD

**Issue by shans** [web-animations/web-animations-js-legacy#141]
In WebKit, the 'transform' property applies to SVG elements. In Firefox it does not. Instead, Firefox SVG elements accept a 'transform' attribute value.

We'll need to detect this feature so that we can decide whether SVG 'transform' animations should be applied through style or through the 'transform' attribute.

**Comment by birtles** [comment]
Yes, that's right. I'm sure the transform property will be added to SVG elements in Firefox in the future but it's not yet. Bear in mind that the values accepted by the property differ to those accepted by the attribute so it's not just a matter of where you plug it in.

**Comment by shans** [comment]
Testing in Firefox 19.0.2 it seems that an explicit style attribute with "transform: rotate(45deg)" _does_ work. Given this, are there any problems with us using the style approach (i.e. are we OK with not targeting non-head for evergreen browsers)?
